ts/src/alpaca.ts

```diff
@@ -444,8 +441,7 @@ export default class alpaca extends Exchange {
         if (since !== undefined) {
             request['start'] = this.yyyymmdd (since);
         }
-        const method = this.safeString (this.options, 'fetchOHLCVMethod', 'cryptoPublicGetCryptoBars');
-        const response = await this[method] (this.extend (request, params));
+        const response = await this.cryptoPublicGetCryptoBars (this.extend (request, params));
```

Comment:
@sc0Vu we have to keep the ability of using `cryptoPublicGetCryptoLatestBars` as well

Reply:
@carlosmiei it seems `cryptoPublicGetCryptoLatestTrades` and `cryptoPublicGetCryptoLatestBars` were removed, and I can't find in their documentation neither.

Reply:
@sc0Vu ok I will check it